marklogic-data-hub-central/ui/src/components/record-view/record-view.tsx

```diff
@@ -133,6 +134,34 @@ const RecordCardView = (props) => {
     return linkObject;
   };
 
+  const download = async (docUri) => {
+    try {
+      const response = await axios({
```

Comment:
Same comment as above. Moving this axios call to /api folder would make it reusable at both the places.